pom.xml

```diff
@@ -80,6 +72,10 @@
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>2.19.1</version>
                 <configuration>
+                    <systemPropertyVariables>
```

Comment:
Can this be done from the unit tests? Otherwise, it won't work if you run them directly via IDE.

Reply:
Setting the system properties does not work with maven build. I'll look for a solution that works on both cases

Reply:
Maybe check in the unit test if it's set, and if not, set it?

Reply:
with maven integration in my IDE it works without any change. Do you think it's not enough?

Reply:
It would be nice to be more flexible, but not a big deal.